layouts/pricing/index.html

```diff
@@ -0,0 +1,499 @@
+            <li>
+              Basic technical and compliance support (during business hours)
+            </li>
+            <li>
+              Documentation/tutorial material
+            </li>
+            <li>
+              Continuous platform-level scanning and security updates
+            </li>
+          </ul>
+        </div>
+
+        <div class="usa-width-one-half">
+          <p>
+            Access to free services
+          </p>
+
+          <div>
+            <ul class="column-single">
+              <li>
+                Postgres database
+              </li>
+              <li>
+                MySQL database
+              </li>
+              <li>
+                Elasticsearch RESTful search and analytics
+              </li>
+              <li>
+                Redis in-memory data store
+              </li>
+              <li>
+                S3 object store
+              </li>
+            </ul>
+            <!-- <ul class="column-single add-on">
+
+              <li>
+                <strong>AWS-RDS</strong>: Persistent, relational databases backed by Amazon RDS
+              </li>
+              <li>
+                <strong>Elasticsearch24</strong>: Elasticsearch 2.4 RESTful search and analytics engine
+              </li>
+              <li>
+                <strong>Elasticsearch56</strong>: Elasticsearch 5.6 RESTful search and analytics engine
+              </li>
+              <li>
+                <strong>Identity-Provider</strong>: Manage client credentials for authenticating cloud.gov users in your app
+              </li>
+              <li>
+                <strong>MongoDB36</strong>: MongoDB 3.6 service for application development and testing
+              </li>
+              <li>
+                <strong>Redis28</strong>: An open source in-memory data structure store
+              </li>
+              <li>
+                <strong>Redis32</strong>: An open source in-memory database
+              </li>
+              <li>
+                <strong>S3</strong>: Amazon S3 provides developers with secure, durable, highly-scalable object storage
+              </li>
+              <li>
+                <strong>Service-Account</strong>: Manage service accounts with access to your cloud.gov organization
+              </li>
+
+            </ul> -->
+
+          </div>
+
+        </div>
+      </section>
+
+      <section class="usa-grid">
+
+        <h2>Paid packages also include:</h2>
+
+        <div class="usa-width-one-half">
+          <ul class="column-single">
+              <li>
+                Self-service management of spaces and user access
+              </li>
+              <li>
+                Single-sign-on with your SAML identity provider
+              </li>
+              <li>
+                Access to premium add-on services
+              </li>
+          </ul>
+        </div>
+
+        <div class="usa-width-one-half">
+          <p>
+            Access to premium services
```

Comment:
Similarly, we should move `Access to premium services` to the bottom of the column to the left, and just make this text `Premium services`.

Reply:
This probably needs a text line after the title, something to approximate the `We aren’t yet charging for paid services. See managed services for details.` item in the existing page.

Reply:
How about `Available free for a limited time`? That's pretty explicit: Paying gets you access, but you're not charged extra... yet!

Reply:
@mogul That sounds like a plan: `Available free for a limited time`